src/components/icon/assets/arrow_down.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16"  xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe there should also be an arrow up for toggling sorting order. But ok we can put these details later

Reply:
May be, but it is all I have at moment.

![image](https://user-images.githubusercontent.com/3184538/69801857-cd350b80-120a-11ea-9b43-1239a82de5b4.png)

Reply:
Ok. so if sorting order changes the arrow remains the same right? We can fix that in some next iteration